package.json

```diff
@@ -47,28 +52,34 @@
     "stylint": "1.5.9"
   },
   "dependencies": {
-    "@atlaskit/editor-core": "134.0.1",
+    "@atlaskit/editor-core": "acezard/editor-core-fork#134.0.1",
+    "@atlaskit/icon": "21.1.0",
     "@atlaskit/media-core": "32.1.2",
-    "@atlaskit/media-editor": "38.1.4",
+    "@atlaskit/media-editor": "^38.1.0",
     "@atlaskit/smart-card": "14.6.1",
+    "@babel/preset-env": "^7.14.7",
+    "@babel/preset-typescript": "^7.14.5",
+    "@babel/runtime": "^7.0.0",
     "@material-ui/core": "4",
     "@material-ui/styles": "4.10.0",
+    "classnames": "^2.2.5",
     "cozy-bar": "7.7.8",
     "cozy-client": "23.1.2",
     "cozy-device-helper": "1.8.0",
     "cozy-doctypes": "1.70.0",
     "cozy-flags": "2.1.0",
     "cozy-realtime": "3.13.0",
-    "cozy-scripts": "5.1.1",
+    "cozy-scripts": "acezard/cozy-scripts-ts",
```

Comment:
Like for `@atlaskit/editor-core` we should use a tag instead of the repository's master branch. Sorry I should have doubled my previous comment.

Reply:
Instead of using a custom fork we should merge the work done in cozy scripts wait for the automatic release and use this new version here!

Reply:
@Crash-- the merge request in cozy-scripts, while working, is still a WIP in my mind. Should we merge it like that or should I work on finishing it 100%? Which is not so trivial because it requires ESLINT to be bumped to latest version which is a huge breaking change for every cozy app

Reply:
Have you updated the corresponding PR with what remains to be done and why we need to upgrade eslint?

Reply:
Same as @Ldoppea since there is work to do, let's publish cozy-scripts on a new tag and use it. We should not rely on personal stuff.

Reply:
the PR is here: https://github.com/cozy/create-cozy-app/pull/1403

It doesn't handle eslint or typechecking, only compiling. It's too difficult with eslint5 and even if you get it to work, lots of typescript features aren't recognized (like interfaces). And for typechecking, I think it's too early to add TSC to cozy-scripts